src/ngraph_encapsulate_op.cc

```diff
@@ -371,7 +393,7 @@ class NGraphEncapsulateOp : public OpKernel {
         m_lru.remove(signature);
         m_lru.push_front(signature);
       }
-      ng_function = it->second;
+      ng_exec = it->second;
```

Comment:
In this case, ng_function is not updated here, but used later
https://github.com/NervanaSystems/ngraph-tf/pull/443/files#diff-4d55399542be3550bdfca644b849afa2R486